src/SignhostAPIClient/Rest/SignHostApiClient.cs

```diff
@@ -74,13 +75,24 @@ public static void RegisterVerification<T>()
 		}
 
 		/// <inheritdoc />
-		public async Task<Transaction> CreateTransactionAsync(Transaction transaction)
+		public async Task<Transaction> CreateTransactionAsync(
+			Transaction transaction)
+			=> await CreateTransactionAsync(transaction, default)
+				.ConfigureAwait(false);
+
+		/// <inheritdoc />
+		public async Task<Transaction> CreateTransactionAsync(
+			Transaction transaction,
+			CancellationToken cancellationToken = default)
 		{
 			if (transaction == null) {
 				throw new ArgumentNullException(nameof(transaction));
 			}
 
-			var result = await client.PostAsync("transaction", JsonContent.From(transaction))
+			var result = await client.PostAsync(
```

Comment:
Since we're adding the cancellationtokens to most of the HttpClient calls I think this would be more readable:

```
var result = await client
	.PostAsync(
		"transaction",
		JsonContent.From(transaction),
		cancellationToken)
	.EnsureSignhostSuccessStatusCodeAsync()
	.ConfigureAwait(false);
```

Reply:
This way we also fix incorrect indentations like at line 293.